src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Markup/KnownTypes.cs

```diff
@@ -3676,6 +3680,10 @@ internal static short GetKnownPropertyAttributeId(KnownElements typeID, string f
                     if (string.Equals(fieldName, "XmlSerializer", StringComparison.Ordinal))
                         return (short)KnownProperties.XmlDataProvider_XmlSerializer;
                     break;
+                case KnownElements.Setter:
```

Comment:
It would be great if we had a category of values that we should test here. For eg: Valid Values, Invalid values. What happens in cases of complex property values i.e SolidColorBrush as a value.

Reply:
I have tested setting following values through Setter -

1. Primitive Values: 

- Boolean (true, false)
- Numeric (integers, doubles)

2. Brushes:

- SolidColorBrush
- LinearGradientBrush
- RadialGradientBrush
- ImageBrush

3. Colors:

- Color values

4. DynamicResource:

- Values using dynamic resource references

5. StaticResource:

- Values using static resource references

6. Bindings:

- Values using data binding

7. Collections:

- Trigger

8. Geometries:

- PathGeometry
- EllipseGeometry

9. Controls:

- Complex control types like ControlTemplate, DataTemplate

10. Transformations:

- RotateTransform
- ScaleTransform
- SkewTransform
- TranslateTransform
- TransformGroup

11. Animations:

- DoubleAnimation

12. Enumerations:

- Enumerated types like HorizontalAlignment, VerticalAlignment

Reply:
Great. Thanks.